src/libNode/ViewChangeBlockProcessing.cpp

```diff
@@ -238,6 +238,18 @@ bool Node::ProcessVCBlockCore(const VCBlock& vcblock) {
 
   UpdateDSCommiteeCompositionAfterVC(vcblock, *m_mediator.m_DSCommittee);
 
+  if (LOOKUP_NODE_MODE) {
+    LOG_STATE("[VCBLK] DS = " << vcblock.GetHeader().GetViewChangeDSEpochNo()
+                              << " Tx = "
+                              << vcblock.GetHeader().GetViewChangeEpochNo());
+    LOG_STATE("[VCBLK] Leader = " << vcblock.GetHeader()
+                                         .GetCandidateLeaderNetworkInfo()
+                                         .GetPrintableIPAddress());
```

Comment:
No need GetPrintableIPAddress(). Just print the Peer object directly. It will be <IP:PORT>